meilisearch/landing
# Add investors block

# Pull Request

## Related issue
Closes #481

## What does this PR do?
- Addition of new icons (investor's logos)
- Creation of the new `Investors` block
- Addition of the corresponding story in Storybook

Some more notes: 
- The `Lorem Ipsum` text is normal (to be updated later)
- I added another `Other block to come` separator to see the background-color differences between each block

## How to test

Go to [the deploy](https://landing-git-addinvestorblock-meili.vercel.app/about-us) and check the block
You can also `yarn storybook` and go to [the story](http://localhost:6006/?path=/story/blocks-investors--default)

## PR checklist
Please check if your PR fulfills the following requirements:

Thank you so much for contributing to Meilisearch!
